packages/console/src/cloud/pages/Main/AutoCreateTenant/index.tsx

```diff
@@ -0,0 +1,23 @@
+import { useContext, useEffect } from 'react';
+
+import { useCloudApi } from '@/cloud/hooks/use-cloud-api';
+import AppLoading from '@/components/AppLoading';
+import { TenantsContext } from '@/contexts/TenantsProvider';
+
+export default function AutoCreateTenant() {
+  const api = useCloudApi();
+  const { appendTenant, tenants } = useContext(TenantsContext);
```

Comment:
Nit: do we need to add an `useTenants` hook to simplify this usage?

Reply:
how?